<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVisit</class>
 <widget class="QDialog" name="AddVisit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Añadir Consulta</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>255</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QFormLayout" name="citaLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="fechaConsultaLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Datos de la cita</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fechaConsultaDateLabel">
        <property name="text">
         <string>Fecha Consulta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="fechaConsultaDateEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ritmoLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ritmo</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="ritmoComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sinusal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flutter | TAQ Auricular</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Otros</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="gradoFuncionalLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Grado Funcional</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="gradoFuncionalComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>II</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>III</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IV</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="fRCVLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>FRCV</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QListWidget" name="listWidgetFRCV">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>120</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <item>
         <property name="text">
          <string>HTA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tabaco</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="parametrosLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="parametrosAnalTicosLabel">
        <property name="text">
         <string>Parametros Analíticos</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="creatininaLabel">
        <property name="text">
         <string>Creatinina</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="creatininaLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="hBLabel">
        <property name="text">
         <string>HB</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="hBLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="fGLabel">
        <property name="text">
         <string>FG</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="fGLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="proBNPLabel">
        <property name="text">
         <string>ProBNP</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="proBNPLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="potasioLabel">
        <property name="text">
         <string>Potasio</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="potasioLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="cirugiaLabel">
        <property name="text">
         <string>Cirugia</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="indicacionQrLabel">
        <property name="text">
         <string>Indicacion Qr</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="indicacionQrComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Si</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="euroScoreLabel">
        <property name="text">
         <string>EuroScore</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="euroScoreLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="tratamientpLabel">
        <property name="text">
         <string>Tratamiento</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="listWidgetTratamiento">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <item>
         <property name="text">
          <string>Diureticos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IECAS | ARA II</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AntiALD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Digox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Antiagregantes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Anticoagulantes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ninguno</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>1051</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>588</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonGuardar">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonGuardar</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
  <tabstop>fechaConsultaDateEdit</tabstop>
  <tabstop>ritmoComboBox</tabstop>
  <tabstop>gradoFuncionalComboBox</tabstop>
  <tabstop>listWidgetFRCV</tabstop>
  <tabstop>hBLineEdit</tabstop>
  <tabstop>creatininaLineEdit</tabstop>
  <tabstop>fGLineEdit</tabstop>
  <tabstop>proBNPLineEdit</tabstop>
  <tabstop>potasioLineEdit</tabstop>
  <tabstop>indicacionQrComboBox</tabstop>
  <tabstop>euroScoreLineEdit</tabstop>
  <tabstop>listWidgetTratamiento</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
